### PR TITLE
Fix: memory leak #6

### PR DESCRIPTION
**Pull request by yanamura**
Fixed this issue
https://github.com/SwiftBond/Bond-Demo-App/issues/6

### CODE REVIEW

File: BondDemo/ListViewController.swift [hunk 1]
```diff
@@ -29,7 +29,7 @@ class ListViewController: UITableViewController {
     // create a bond for table view data source
 
     // establish a bond between view model and table view
-    dataSource.bindTo(tableView) { (indexPath, dataSource, tableView) -> UITableViewCell in
+    dataSource.bindTo(tableView) { [unowned self] (indexPath, dataSource, tableView) -> UITableViewCell in
```
Comment (srdanrasic) [inline BondDemo/ListViewController.swift]: Do we even need `self` if `tableView` is passed in the closure?

Comment (yanamura) [inline BondDemo/ListViewController.swift]: no need self, thanks

**Comment by yanamura** [comment]
I removed unnecessary self.

**Comment by srdanrasic** [comment]
Thanks for your contribution! :)

**Pull request merged**